Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/ViewportMessages.h

```diff
@@ -347,20 +341,39 @@ namespace AzToolsFramework
 
     //! Wrap EBus call to retrieve manipulator line bound width.
     //! @note It is possible to pass AzFramework::InvalidViewportId to perform a Broadcast as opposed to a targeted Event.
-    inline float ManipulatorLineBoundWidth(AzFramework::ViewportId viewportId)
+    inline float ManipulatorLineBoundWidth(const AzFramework::ViewportId viewportId)
     {
         float lineBoundWidth = 0.0f;
         if (viewportId != AzFramework::InvalidViewportId)
         {
-            ViewportInteraction::ViewportInteractionRequestBus::EventResult(
-                lineBoundWidth, viewportId, &ViewportInteraction::ViewportInteractionRequestBus::Events::ManipulatorLineBoundWidth);
+            ViewportInteraction::ViewportSettingsRequestBus::EventResult(
+                lineBoundWidth, viewportId, &ViewportInteraction::ViewportSettingsRequestBus::Events::ManipulatorLineBoundWidth);
         }
         else
         {
-            ViewportInteraction::ViewportInteractionRequestBus::BroadcastResult(
-                lineBoundWidth, &ViewportInteraction::ViewportInteractionRequestBus::Events::ManipulatorLineBoundWidth);
+            ViewportInteraction::ViewportSettingsRequestBus::BroadcastResult(
+                lineBoundWidth, &ViewportInteraction::ViewportSettingsRequestBus::Events::ManipulatorLineBoundWidth);
         }
 
         return lineBoundWidth;
     }
+
+    //! Wrap EBus call to retrieve manipulator line bound width.
+    //! @note It is possible to pass AzFramework::InvalidViewportId to perform a Broadcast as opposed to a targeted Event.
+    inline float ManipulatorCicleBoundWidth(const AzFramework::ViewportId viewportId)
```

Comment:
Large function for `inline`, it's eventually the compiler's decision to make it inline or not. Is it possible to move to a cpp?

Reply:
Yes but it's just really an EBus wrapper - I can move it though there's just annoyingly no `ViewportMessages.cpp` right now

Reply:
Sounds good to me to add a `ViewportMessages.cpp`. Please create a task  for it.